### ISSUE TITLE
Template Opens Over Existing File

### ISSUE THREAD

**Issue by klhes** [sebbrudzinski/Open-LaTeX-Studio#115]
Steps to Reproduce
1. Select File -> Open File
2. Select a .tex file and Open
3. Select File -> Open Template
4. Select Plain Cover Letter and click Load Template

Actual Result
The template content overwrites the open document's content. If the user saves the document, they will lose the original content.

Expected Result
When a template is opened, the user should be asked if they would like to save the current document (this is addressed in [Issue #100](https://github.com/sebbrudzinski/Open-LaTeX-Studio/issues/100)) and a new .tex document should open.

**Comment by aQaTL** [comment]
I figured it out: 
-when you opens new file, program calls method `setCurrentFile` from EditorTopComponent,
-it allows program to store info which file is currently opened,
-but, for now template is being loaded with this code: 
`etc.setEditorContent(FileService.readFromStream(getClass().getResourceAsStream(templatesList.getSelectedValue().getPath())));`
-it just changes text in editor area,
-so, EditorTopComponent still have currentFile set to file that you opened before,
-so, when you tires to save file, it gets file path from EditorTopComponent from currentFile variable.

I hope, that it's understandable. :)

I fixed it, but couldn't (or don't know how) open another pull request without changes in Issue #100, so I just committed them to my currently opened [pull request](https://github.com/sebbrudzinski/Open-LaTeX-Studio/pull/104). Sorry for messing up.
